repos/system_upgrade/common/libraries/rhui.py

```diff
@@ -146,6 +145,22 @@
                 ('leapp-azure.repo', YUM_REPOS_PATH)
             ],
         },
+        # FIXME(mhecko): This entry is identical to the azure one, since we have no EUS content yet, therefore, it
+        # #              serves only the purpose of containing the name of rhui client package to correctly detect
+        # #              cloud provider. Trying to work around this entry by specifying --channel, will result in
+        # #              failures - there is no repomapping for EUS content, and the name of target pkg differs on EUS.
+        'azure-eus': {
+            'src_pkg': 'rhui-azure-rhel8-eus',
+            'target_pkg': 'rhui-azure-rhel9',
```

Comment:
Let's comment that EUS mapping will be added later when the EUS image is released. In case such an image is available sooner than we release a new Leapp version, the switch can be done using official MS docs.

Reply:
The switch cannot be done since I presume the target package will be different. We can either predict the name of target package, but make the EUS 8to9 upgrade unavailable, or we can keep it as it is, which would mean that we have to patch this piece of code when the EUS image is available.

Reply:
It can be done after the upgrade:
https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-rhui#switch-a-rhel-vm-8x-to-eus-version-lock-to-a-specific-minor-version

Reply:
I have added the comment with the most recent push